src/_includes/templates/nav/about-us-media.html

```diff
@@ -1,9 +1,9 @@
 <a href="/the-bureau/">
     <img src="/static/img/nav-about-us-video.png"
          alt="About Us video image" />
-    <h4 class="short-desc u-mt15">
+    <p class="h4 short-desc u-mt15">
```

Comment:
Then the first heading of the page is an h4, which is an error in the traditional outline.